### ISSUE TITLE
Train.py loss incorrectly calculated

### ISSUE THREAD

**Issue by JackBeerman** [baeklab/PARCtorch#24]
train.py has  # Compute loss
            loss = criterion(predictions[:, :, 1:, :, :], gt[:, :, 1:, :, :])
this was originally used in the burgers demo to skip the reynolds number. However when you are making the prediction for EM you will not include Temperature.

**Comment by chengxinlun** [comment]
Feel free to make any necessary changes since you added the file Jack. There should be no breaking changes on my end.

I have not used any functionalities provided by this module.

**Comment by JackBeerman** [comment]
It depends if we want to create separate training logic for the application at hand or just require to users to create their own training function?

**Comment by JackBeerman** [comment]
This is fixed and documented in train_model.py
